src/sentry/coreapi.py

```diff
@@ -693,6 +693,25 @@ def validate_data(self, project, data):
                 })
                 del data['environment']
 
+        if data.get('time_spent'):
+            try:
+                data['time_spent'] = int(data['time_spent'])
+            except (ValueError, TypeError):
+                data['errors'].append({
+                    'type': EventError.INVALID_DATA,
+                    'name': 'time_spent',
+                    'value': data['time_spent'],
+                })
+                del data['time_spent']
+            else:
+                if data['time_spent'] > 2147483647:  # Max value for the int4 column
```

Comment:
can you write this as `0x7fffffff`? Makes it much more obvious that this is a specific value.